test/powershell/Modules/Microsoft.PowerShell.Utility/Set-Date.Tests.ps1

```diff
@@ -12,7 +13,7 @@ Describe "Set-Date for admin" -Tag @('CI', 'RequireAdminOnWindows', 'RequireSudo
     }
 }
 
-Describe "Set-Date" -Tag 'CI' {
+Describe "Set-Date" -Tag @('CI', 'RequireNoSudoOnUnix') {
```

Comment:
This tag is not needed for this test. Only tests with `RequireSudoOnUnix` tag will be included in a SUDO test run.
I think this new tag should be removed from `build.psm1` too.

Reply:
Yes, please remove this tag

Reply:
Without this tag, this test is run when `sudo` is used and this test fails because it's expecting an exception, but no exception occurs.

Reply:
We should only run `REQUIRESUDOONUNIX` tests with -sudo.  I have a fix for travis.ps1 for this issue.

Reply:
cc @JamesWTruher

Reply:
@TravisEz13 that would make sense, so then we should exclude all tests that don't have `RequiresSudoOnUnix` when `-Sudo` is used, correct?

Reply:
when running sudo, only include the tests with the `RequiresSudoOnUnix` Tag.

Reply:
Ok, I think I got it figured out, but it means that `RequireSudoOnUnix` tag means `CI`,`Feature`,`Slow`,etc... no longer apply.